app/src/org/commcare/activities/AppManagerActivity.java

```diff
@@ -47,6 +48,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        Log.d("Shivamm", "Should give lint issues");
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/b67bf14163879f9fb09208858b78318e65c45843/app/src/org/commcare/activities/AppManagerActivity.java#L51"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>